word-guess-classes.rb

```diff
@@ -0,0 +1,235 @@
+require 'colorize'
+
+class Word
+  attr_accessor :guessed_letters, :blanks, :wrong_guesses, :letters, :name
+
+  def initialize(word)
+    @name = word
+    @letters = word.chars
+    @blanks = []
+    @guessed_letters = []
+    @guess = ""
+    @wrong_guesses = 0
+    create_blank_array
+  end
+
+  def create_blank_array
+    @letters.length.times do
+      @blanks << "_"
+    end
+  end
+
+  def print_blanks
+    @blanks.each do | value |
+      print value
+    end
+  end
+
+  def check_guess(guess)
+    if @guessed_letters.include?(guess)
+      puts "\nYou've already guessed this letter.".colorize(:yellow)
```

Comment:
This logic all looks good to me. One observation is that repeat letters get a warning up here, and also count as a wrong guess - not sure if that's what you intended.